app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileScreen.kt

```diff
@@ -208,10 +202,12 @@ private fun UserProfileScreen(
                     userStatus = status,
                     onStatusClicked = onStatusClicked
                 )
-                OtherAccountsList(
-                    otherAccounts = otherAccounts,
-                    onAddAccountClick = onAddAccountClick
-                )
+                if (!state.otherAccounts.isNullOrEmpty()) {
```

Comment:
will it ever be empty ?

Reply:
In fact, it will be empty by default, changed to `isNotEmpty` 👍